src/query/service/src/sessions/session_privilege_mgr.rs

```diff
@@ -242,6 +249,7 @@ impl<'a> SessionPrivilegeManager for SessionPrivilegeManagerImpl<'a> {
         &self,
         object: &GrantObject,
         privilege: UserPrivilegeType,
+        create_object: bool,
```

Comment:
can we derive `create_object` parameter by the `privilege` parameter ?

Reply:
Same like this:https://github.com/datafuselabs/databend/pull/15751/files#discussion_r1630942306